diff --git a/Makefile b/Makefile
index b1be3e0..0d8aaed 100644
--- a/Makefile
+++ b/Makefile
@@ -181,5 +181,4 @@ deploy@production:
 
 #######
 # App #
-####### 
- This line is expected in the patch
\ No newline at end of file
+#######
\ No newline at end of file
diff --git a/ansible/deploy.yaml b/ansible/deploy.yaml
new file mode 100644
index 0000000..9ab9e15
--- /dev/null
+++ b/ansible/deploy.yaml
@@ -0,0 +1,6 @@
+---
+
+- hosts: deploy
+  gather_facts: false
+  roles:
+    - manala.deploy
