diff --git a/Makefile b/Makefile
index b94e9fc..0fe662c 100644
--- a/Makefile
+++ b/Makefile
@@ -114,5 +114,3 @@ deploy@production:
 ##########
 # Custom #
 ##########
- 
- This line is expected in the patch
\ No newline at end of file
diff --git a/ansible/deploy.yml b/ansible/deploy.yml
new file mode 100644
index 0000000..9ab9e15
--- /dev/null
+++ b/ansible/deploy.yml
@@ -0,0 +1,6 @@
+---
+
+- hosts: deploy
+  gather_facts: false
+  roles:
+    - manala.deploy
